### DISCUSSION TITLE
Understand Meaning Of The Nodes

### DISCUSSION THREAD

**Discussion by Last-Winner** [community/community#66637] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

<script>
--
  | (function () {
  | // INITIALIZATION OF CLIPBOARD
  | HSCore.components.HSClipboard.init('.js-clipboard');
  |  
  | let searchAttributes = {
  | 'name': 'title',
  | 'img': 'img',
  | 'rate': 'rate',
  | 'address': 'address',
  | 'link': 'link',
  | 'group': 'group',
  | 'groupid': 'groupid',
  | 'website': 'website',
  | 'is_verified': 'is_checked',
  | 'ps': 'ps',
  | 'desc': 'desc',
  | 'reputation' : 'reputation'
  | }
  |  
  | etherSearchInit({
  | id: 'search-panel',
  | attributes: searchAttributes
  | });
  | })();
  |  
  |  
  | </script>

<script>
  | $(window).on('load', function () {
  | myfn_subscribeNewsletter.startFn("#btnSubscribeNewsletter", "#signupSrEmail")
  | });
  |  
  | $(document).on('ready', function () {
  | // initialization of cubeportfolio
  | $.HSCore.components.HSCubeportfolio.init('.cbp');
  |  
  | const clipboard = new ClipboardJS('.js-clipboard');
  | clipboard.on('success', function (e) {
  | let toolTip = bootstrap.Tooltip.getInstance(e.trigger);
  | var oriTooltip = toolTip._config.originalTitle
  |  
  | toolTip._config.title = 'Copied!';
  | toolTip.setContent();
  | toolTip.update();
  |  
  | setTimeout(function () {
  | toolTip._config.title = oriTooltip;
  | }, 1000);
  | });
  | });
  |  
  | </script>
  | <script type="text/javascript">
  | var sid = '68d66e619b12bcc97e95b3343354b29d';
  | var strGlobal = getCookie("etherscan_switch_age_datetime");
  | var strGlobalFee = sessionStorage.getItem("ShowFee");
  | var cookieconsent = getCookie("etherscan_cookieconsent");
  | var ignoreCopyConfirmation = getCopyAddressConfirmationSetting();
  |  
  | if (cookieconsent !== "True" && document.getElementById("divcookie")) {
  | document.getElementById("divcookie").style.display = "block";
  | };
  |  
  |  
  | function getCookie(cname) {
  | var name = cname + "=";
  | var ca = document.cookie.split(';');
  | for (var i = 0; i < ca.length; i++) {
  | var c = ca[i];
  | while (c.charAt(0) === ' ') {
  | c = c.substring(1);
  | }
  | if (c.indexOf(name) === 0) {
  | return c.substring(name.length, c.length);
  | }
  | }
  | return "";
  | }
  |  
  | $("#btnCookie").click(function () {
  | $("#divcookie").fadeOut("slow", function () {
  | var d = new Date();
  | d.setTime(d.getTime() + (1095 * 24 * 60 * 60 * 1000));
  | var expires = "expires=" + d.toUTCString();
  | document.cookie = "etherscan_cookieconsent=True" + ";" + expires + ";path=/";;
  | });
  | });
  |  
  | function handleSearchText(data) {
  | searchAddress = false;
  | if ($("search-panel").val() == "") {
  | $("#hdnSearchText").val("");
  | $("#hdnSearchLabel").val("");
  | }
  | }
  |  
  | function writeCookie(key, value, days) {
  | var date = new Date();
  | days = days \|\| 365;
  | date.setTime(+ date + (days * 86400000));
  | window.document.cookie = key + "=" + value + "; expires=" + date.toGMTString() + "; path=/";
  | return value;
  | }
  |  
  | function setThemeMode(mode) {
  | writeCookie('displaymode', mode);
  | }
  |  
  | // Note: this function will be called from where need to init the "Copy Address Confirm Modal".
  | function openCopyAddressConfirmModal(address, message) {
  | $("#addressToCopy").text(address);
  | $("#copyAddressConfirmMessage").text(message);
  |  
  | const clipboard = new ClipboardJS("#btnConfirmCopyAddress");
  | document.getElementById('copyAddressConfirmModal').addEventListener('hidden.bs.modal', event => {
  | clipboard.destroy();
  | });
  |  
  | const copyAddressConfirmModal = new bootstrap.Modal('#copyAddressConfirmModal', {
  | backdrop: true
  | })
  | copyAddressConfirmModal.show();
  | }
  |  
  | function saveCopyAddressConfirmationSetting(ignoreCopyConfirmation) {
  | const copyAddressConfirmationSetting = {
  | ignoreCopyConfirmation: ignoreCopyConfirmation,
  | expiresOn: Date.now() + 1000 * 60 * 60 * 24 * 30
  | };
  | localStorage.setItem("copyAddressConfirmationSetting", JSON.stringify(copyAddressConfirmationSetting));
  | }
  |  
  | function getCopyAddressConfirmationSetting() {
  | try {
  | const settings = localStorage.getItem("copyAddressConfirmationSetting");
  | if (settings) {
  | const copyAddressConfirmationSetting = JSON.parse(settings);
 

<br class="Apple-interchange-newline">
return copyAddressConfirmationSetting.ignoreCopyConfirmation && copyAddressConfirmationSetting.expiresOn > Date.now();
--
  | }
  | else {
  | return false;
  | }
  | } catch (e) {
  | console.log(e);
  | return false;
  | }
  | }

**Comment by fathi-riham-mn** [comment]
Help me earn github galaxy brain achievement
